### ISSUE TITLE
Problem prices for chronicles, rennaissance and legends

### ISSUE THREAD

**Issue by gweniche** [mtgjson/mtgjson#787]
Hi,

there is a problem with the prices from card market for the set chronicles, rennaissance and legends, they are not accurate.

Thx

**Comment by ZeldaZach** [comment]
Can you provide more information? What makes you think the prices are inaccurate?

**Comment by gweniche** [comment]
Like this card for example, https://www.cardmarket.com/fr/Magic/Products/Singles/The-Dark/Ball-Lightning, the price in database are too low.

**Comment by kodawah** [comment]
As someone wiser than me said:

We receive all of those fields from Cardmarket, we fetched the 1-day avg, the 7-day avg, and the overall "avg sell price", in that order of preference. Usually the day-1 avg is the best measure of a card's changing value. It's not really MTGJSON's responsibility to prevent people from underselling rares. Or perhaps a lower price would be more appropriate for that rare anyway, if that is what is being listed and sold.

So to put it another way: We have no insight into the sales volume of a card at Cardmarket, we just load the data they provide. It's on them to prevent outlier sales from affecting their average metrics.
